tests/functional_tests/0019_list_snapshot_test.py

```diff
@@ -37,16 +37,28 @@ def common_function(
     jails_as_rows, full=False
 ):
     for flag in SORTING_FLAGS:
-        command = ['snaplist', jail.name, '-s', flag]
+        if type(jail) is list:
```

Comment:
nit: the idiomatic way is to use `isinstance()`
```suggestion
        if isinstance(jail, list):
```